pkgs/development/libraries/olm/default.nix

```diff
@@ -27,5 +27,8 @@ stdenv.mkDerivation rec {
     homepage = "https://gitlab.matrix.org/matrix-org/olm";
     license = licenses.asl20;
     maintainers = with maintainers; [ tilpner oxzi ];
+    knownVulnerabilities = [
+      "libolm is deprecated upstream with an AES cache timing vulnerability that upstream doesn't plan to fix"
```

Comment:
Could you please add more details, for example, linking the olm commit stating it as deprecated - https://gitlab.matrix.org/matrix-org/olm/-/commit/6d4b5b07887821a95b144091c8497d09d377f985 - and Soatok's blog post.

Reply:
Yes, thanks for nudging me to do this. I avoided doing it because I was worried about being sufficiently neutral and describing the potential risks and trade‐offs sufficiently well, but it’s important in a situation like this. I’ve pushed a long but hopefully unbiased and informative explanation of the issues and maintenance status of the package that should let users make an informed decision. Please let me know what you think.

Reply:
I enjoyed reading your deprecation information text. It informs the user with enough details to look into it.

Thanks again for all the work you are putting into this :)